ksqldb-serde/src/test/java/io/confluent/ksql/serde/json/KsqlJsonDeserializerTest.java

```diff
@@ -131,22 +130,16 @@ public class KsqlJsonDeserializerTest {
           .addSerializer(java.sql.Date.class, new EpochDaySerializer())
       );
 
-  @Parameters(name = "{0}")
-  public static Collection<Object[]> data() {
-    return Arrays.asList(new Object[][]{{"Plain JSON", false}, {"Magic byte prefixed", true}});
```

Comment:
Ok, so do we still test both of these options/parameters?

If not, why?

Reply:
Not anymore. These test class was testing JSON and JSON_SR. The new JsonSchemaConverter does not use like 95% of these tests. I created a new `KsqlJsonSchemaDeserializerTest` class for that. I will add tests on it the more I understand JSON_SR.